src/sentry/static/sentry/app/views/onboarding/projectSetup/eventWaiter.tsx

```diff
@@ -55,10 +57,10 @@ class EventWaiter extends React.Component {
     this.stopPolling();
   }
 
-  intervalId = null;
+  intervalId: NodeJS.Timeout | null = null;
```

Comment:
NodeJS?

Reply:
Yeah I don't... know why it's typing like this :/

Reply:
Okay, got around this by using `window.setTimeout`.